pandas/tests/series/methods/test_astype.py

```diff
@@ -471,7 +471,7 @@ class TestAstypeString:
     def test_astype_string_to_extension_dtype_roundtrip(
         self, data, dtype, request, nullable_string_dtype
     ):
-        if dtype == "boolean" or (dtype == "timedelta64[ns]" and NaT in data):
```

Comment:
just for my understanding, why did this one change?

Reply:
ah it's because
```
ser.astype(nullable_string_dtype)
```
introduces `'<NA>'`, and then `.astype('timedelta64[ns]')` couldn't handle that (but it can after this PR!)